homeassistant/components/met/weather.py

```diff
@@ -221,6 +228,11 @@ def forecast(self):
                 for k, v in FORECAST_MAP.items()
                 if met_item.get(v) is not None
             }
+            if not self._is_metric:
+                if ATTR_FORECAST_PRECIPITATION in ha_item:
+                    ha_item[ATTR_FORECAST_PRECIPITATION] = round(
+                        ha_item.get(ATTR_FORECAST_PRECIPITATION) / 25.4, 2
```

Comment:
Please use our distance util to convert the unit.

https://github.com/home-assistant/core/blob/ce3f6c368a023251b6ceb2aa866683bc1450912d/homeassistant/util/distance.py#L52-L53

Please don't use dict.get when we know that the key is in the dict. Use `dict[key]`.

Reply:
I wasn't on the latest version on my dev instance, so I missed the added units in the distance util. Fixed now.